testing/testing.gni

```diff
@@ -80,6 +80,7 @@ template("dart_snapshot_kernel") {
       rebase_path("$root_out_dir/flutter_patched_sdk"),
       "--target",
       "flutter",
+      "--enable-experiment=non-nullable",
```

Comment:
Does this mean that all of the Dart language tests will run in nnbd mode?

Reply:
This is used by the engine unit-tests that have Dart fixtures. The Dart unit-tests still run with the default snapshot generation steps (which don't use strong null-safety). I don't think the snaphotting for the scenarios app uses strong null-safety either.